src/pages/CreateGroup/createGroupSlice.ts

```diff
@@ -0,0 +1,38 @@
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+
+interface FormDataUpdate {
+  fieldName: string;
+  value: string;
+}
+
+interface CreateGroupState {
+  formData: { [key: string]: string };
```

Comment:
Можно так писать, тоже самое, на вкус

```suggestion
  formData: Record<string, string>;
```